src/util/mmap_impl_win32.h

```diff
@@ -155,16 +153,15 @@ Status MappedFile::map(MappedFileFragment *view, size_t offset, size_t size,
                               << ". Error: " << GetLastError();
   }
 
-  const DWORD granuality = get_system_granuality();
+  DWORD granuality = get_system_granuality();
```

Comment:
Shouldn't it remain const?, it is not supposed to be changed further in code

Reply:
I try not to reuse variables as much time as it is possible.
At the same time, other code does not use const in local functions as well.

const T& vs T& for parameters is very important on the other hand.

Reply:
Btw, the result of the get_system_granlarity call was not used in your patch and I didn't see it.

Reply:
Uh... looking at code yeah... It seems I moved result of function to variable, but forgot to remove invocation https://github.com/ku-nlp/jumanpp/commit/0be5fa004d145f05c07f5185effa697403fda454#diff-ebcfd21b2ea62c81d687e6e35fdc99aeR158

Really lame of me

Reply:
To be honest Im not really sure it is good idea to re-use variable as much as possible.
Minimizing variable scope and making them const makes it easier to reason about code which consequently makes it easier to understand code.

But this is just my opinion, if you'd like to keep it like that it is ok too.

Reply:
**not to reuse**.
I agree that reusing should be done only when its needed.
Register allocators are pretty smart to get the score right.

It's just... the other code here doesn't use const for locals.